Dockerfile.docs

```diff
@@ -1,31 +1,19 @@
 # DOCKER STAGE: BUILD
 FROM node:11.4-alpine as builder
 
-ARG NODE_ENV=production
-ENV NODE_ENV $NODE_ENV
+ENV NODE_ENV=production
 
 ## Install build toolchain, install node deps and compile native add-ons
 RUN apk add --no-cache python make g++
 
-RUN npm set progress=false
-RUN npm set unsafe-perm true
-
-RUN npm config set depth 0
-
-## Use app/ subdir so root tsconfig can be extended
-RUN mkdir -p ./app
-COPY docs/. ./app/
-COPY tsconfig.json .
```

Comment:
Is there not a need to have the base tsconfig.json copied over any more?

Reply:
oops. nice catch